app/assets/stylesheets/marvinjs.scss

```diff
@@ -39,6 +39,7 @@
     .file-save-link {
       flex-shrink: 0;
       margin: 0 20px 0 0;
+      cursor: pointer;
```

Comment:
It must inherit this attribute from `.btn` class.
![Screenshot from 2021-10-05 12-36-36](https://user-images.githubusercontent.com/47317017/136007645-4e1cf409-90b4-4097-af3a-4e25d14dc1bd.png)

Reply:
Thanks!